mockcache.py

```diff
@@ -348,23 +354,21 @@ def check_key(key, key_extra_len=0):
         Is an unicode string (Raises MockcachedStringEncodingError)
         Is None (Raises MockcachedKeyNoneError)
     """
-    import types
-    if type(key) == types.TupleType:
+    if type(key) == tuple:
         key = key[1]
     if not key:
-        raise Client.MockcachedKeyNoneError, ("Key is None")
+        raise Client.MockcachedKeyNoneError("Key is None")
     if isinstance(key, unicode):
```

Comment:
Oh, I left a wrong comment before:

> Doesn’t `unicode` exist in Python?

What I wanted to say is `unicode` is no more available since Python 3:

```python
>>> import sys
>>> print(sys.version)
3.5.0 (default, Nov 17 2015, 01:59:41) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)]
>>> unicode
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'unicode' is not defined
```